x/nft/base/handler_test.go

```diff
@@ -0,0 +1,274 @@
+package base
+
+import (
+	"testing"
+
+	"github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/store"
+	"github.com/iov-one/weave/x"
+	"github.com/iov-one/weave/x/nft"
+	"github.com/iov-one/weave/x/nft/blockchain"
+	"github.com/iov-one/weave/x/nft/username"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestApprovalOpsHandler(t *testing.T) {
+	Convey("Test approval ops handler", t, func() {
+		userId := []byte("user")
+		user2Id := []byte("user2")
+		user3Id := []byte("user3")
+		user4Id := []byte("user4")
```

Comment:
pp: can you be a bit more verbose with the names? For example
* userId  = bobsName
* aliceNameWithBobApproved 
* bobsNameWithAliceApproved 
* bobsNameWithAliceApprovedImmutable